File: app/src/main/java/com/cookplan/recipe/edit/add_ingredients/EditRecipeIngredientsView.java
package com.cookplan.recipe.edit.add_ingredients;

import com.cookplan.models.Ingredient;

import java.util.List;

/**
 * Created by DariaEfimova on 21.03.17.
 */

public interface EditRecipeIngredientsView {

    void setErrorToast(String error);

    void setIngredientList(List<Ingredient> ingredientList);
}

File: app/src/main/java/com/cookplan/recipe/edit/ingredients/EditRecipeIngredientsView.kt
package com.cookplan.recipe.edit.ingredients

import com.cookplan.models.Ingredient

/**
 * Created by DariaEfimova on 21.03.17.
 */

interface EditRecipeIngredientsView {

    fun setErrorToast(errorMessage: String)

    fun setErrorToast(errorId: Int)

    fun setIngredientList(ingredientList: List<Ingredient>)
}
